config/src/schemas/heas.schema.ts

```diff
@@ -1,13 +1,125 @@
 import type {
   ActivityConfig,
-  ContentContainer,
+  ContentContainerConfig,
   Schema,
 } from '@tailor-cms/interfaces/schema';
+import type { ContentContainer } from '@tailor-cms/interfaces/activity';
 import { ContentContainerType } from '@tailor-cms/content-container-collection/types.js';
+import type { ContentElement } from '@tailor-cms/interfaces/content-element';
+import { ContentElementType } from '@tailor-cms/content-element-collection/types.js';
 import { MetaInputType } from '@tailor-cms/meta-element-collection/types.js';
+import type { Repository } from '@tailor-cms/interfaces/repository';
 
 import { DEFAULT_WORKFLOW } from '../workflows/default.workflow';
 
+const guidelines = (
```

Comment:
Consider adding few words into the changelog / PR desc about the feature. We should also create follow-up ticket for documenting it.